### PR TITLE
Update LOCALIZATION_CONTRIBUTORS

### PR DESCRIPTION
**Pull request by viniciusbribeiro**
Adding myself as a contributor for Portuguese

### Summary

Related #<replace_me_with_related_issue_number>

### CODE REVIEW

File: LOCALIZATION_CONTRIBUTORS [hunk 1]
```diff
@@ -61,7 +61,7 @@ Mauricio Caldeira Baptista
 Natália Capra Ferrazzo
 André Galdino
 Marco Antônio Guimarães Auad Barroca
-
+Vinicius Barboza Ribeiro
```
Comment (omarcostahamido) [inline LOCALIZATION_CONTRIBUTORS:64]: Please do still leave 2 blank lines at the end to separate from the next language group.
Thanks

**Review by omarcostahamido — CHANGES_REQUESTED** [review]

**Comment by omarcostahamido** [comment]
Joined PT_UN translations in Crowdin

**Review by omarcostahamido — APPROVED** [review]
LGTM

File: LOCALIZATION_CONTRIBUTORS (hunk 1 above)
Comment (viniciusbribeiro) [reply LOCALIZATION_CONTRIBUTORS:64]: Sent a new edit request

Comment (omarcostahamido) [reply LOCALIZATION_CONTRIBUTORS:64]: I've fixed this myself 13 minutes ago.
Thank you. It should be on its way to be merged soon now ;)

Comment (viniciusbribeiro) [reply LOCALIZATION_CONTRIBUTORS:64]: Oh, my bad, did not catch that. I'm just getting the hang of this, but I'm eager to contribute. (:

Comment (omarcostahamido) [reply LOCALIZATION_CONTRIBUTORS:64]: No problem. I'm happy to know that you want to join us and that you made the PR together with the crowdin request at the same time, following the instructions.

**Review by SooluThomas — APPROVED** [review]

**Comment by omarcostahamido** [comment]
thank you @SooluThomas

**Pull request merged**